dockers/docker-fpm-frr/Dockerfile.j2

```diff
@@ -13,7 +13,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
 
 # Install required packages
-RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4 libc-ares2 iproute libpython2.7
+RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4 libc-ares2 iproute libpython2.7 libjson-c2 logrotate
```

Comment:
is logrotate required inside the docker?

Reply:
@nikos-github , is logrotate started inside the docker? if not, can you remove the logrotate as a dependency from frr package?

Reply:
@stcheng Yes.
@lguohan It's not started.

In case it wasn't noticed, the exact same dependency exists in quagga debian pkg.

Furthermore, both packages that I added, exist today with frr 4.0 inside the docker:

ii  libjson-c2:amd64                   0.11-4                 amd64                  JSON manipulation library - shared library
ii  logrotate                           3.8.7-1+b1                amd64                  Log rotation utility

But I'm making the change explicit rather than have this:

Step 35/47 : RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; dpkg_apt /debs/frr_6.0.2-0~sonic.debian8+1_amd64.deb
 ---> Running in 3bbd7c286890
Selecting previously unselected package frr.
(Reading database ... 10403 files and directories currently installed.)
Preparing to unpack .../frr_6.0.2-0~sonic.debian8+1_amd64.deb ...
Unpacking frr (6.0.2-0) ...
Processing triggers for systemd (215-17+deb8u8) ...
dpkg: dependency problems prevent configuration of frr:
 frr depends on libjson-c2 (>= 0.11); however:
  Package libjson-c2 is not installed.
 frr depends on logrotate (>= 3.2-11); however:
  Package logrotate is not installed.

dpkg: error processing package frr (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 frr
Reading package lists...
Building dependency tree...
Reading state information...
Correcting dependencies... Done
The following extra packages will be installed:
  cron libjson-c2 libpopt0 logrotate